index.js

```diff
@@ -1,10 +1,7 @@
-var VOWELS   = /[aeiou]/gi;
-var VOWELS_AND_SPACE = /[aeiou\s]/g;
-
 exports.parse = function parse(string, join){
   if(typeof string !== 'string'){
     throw new TypeError('Expected a string as the first option');
   }
 
-  return string.replace(join ? VOWELS_AND_SPACE : VOWELS, '');
+  return string.replace(join ? /[aeiou\s]/gi : /[aeiou]/gi);
```

Comment:
Missing the second parameter here, running this will replace all vowels with 'undefined'.